tests/fixtures/test_data.py

```
@@ -0,0 +1,374 @@
+SYSTEM_INFO_NO_ZFS = {
+    'hostname': 'test-ubuntu',
+    'docker_version': '20.10.21',
+    'zfs_available': False,
+    'zfs_version': None,
+    'compose_base': '/home/user/compose',
+    'appdata_base': '/home/user/appdata',
+    'zfs_pool': None
+}
+
+# Sample migration requests
+from backend.models import MigrationRequest
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)